src/setup.c

```diff
@@ -120,6 +123,54 @@ static void setup_usart(void)
 	usart_enable(USART3);
 }
 
+/**
+ * @brief Setup SPI for gyroscope communication.
+ *
+ * SPI operational features of the gyroscope:
+ *
+ * - Data is delivered MSB first and LSB last.
```

Comment:
I would add this first paragraph on documentation section

Reply:
Yeah, good idea. Moved that to the documentation. :blush: